test/test-tcp-open.c

```diff
@@ -122,6 +124,21 @@ static void read_cb(uv_stream_t* tcp, ssize_t nread, const uv_buf_t* buf) {
 }
 
 
+static void read1_cb(uv_stream_t* tcp, ssize_t nread, const uv_buf_t* buf) {
+  ASSERT(tcp != NULL);
+
+  if (nread >= 0) {
+    for (int i = 0; i < nread; ++i)
```

Comment:
Can you use `for (i = ...; ...; ...)` here?  I'm reasonably sure `int i = ...` will be rejected by compilers in strict C89 mode.

Reply:
fixed